README.md

```diff
@@ -200,7 +275,7 @@ Acme Gifts provides the ability for users to login using Twitter credentials as
 Follow the steps below to configure an existing twitter account for Acme Gifts.
 
 1. Specify valid twitter generated keys/secrets/tokens. 
-   These properties to update are located under the "Optional Microservice Configuration" label in the "properties" section of sample-acmegifts/pom.xml. Update the CHANGE_ME values with valid twitter generated keys/secrets/tokens.
+   These properties to update for Maven are located under the "Optional Microservice Configuration" label in the "properties" section of sample-acmegifts/pom.xml and, for Gradle, in the ExtraPropertiesExtension (ext) of sample-acmegifts/build.gradle. Update the CHANGE_ME values with valid twitter generated keys/secrets/tokens.
```

Comment:
This first sentence does not flow well. I suggest the following:
For Maven, the properties to update are located under the "Optional Microservice Configuration" label in the "properties" section of sample-acmegifts/pom.xml. For Gradle, the properties are located in the ExtraPropertiesExtension (ext) of sample-acmegifts/build.gradle.